packages/x-components/src/design-system/utilities/border-width.scss

```diff
@@ -0,0 +1,34 @@
+@import '../mixins';
+
+*[class*='x-border-width--'] {
+  border: 0;
```

Comment:
What is this for? I think it is breaking combining border width and color utilities. Using border-width here seems to fix the issue, but I don't know if it is breaking something else

Reply:
This is to reset the default browser styles, with a width of 3px;
Tailwind resets to all elements with `*` selector. 
By default an element doesn't have border, even with `x-border-color`. You have to add also a width.

If I remove this I get this:
![image](https://user-images.githubusercontent.com/5759712/133612426-70a772d0-2f6d-4ad0-a096-b21a525a5fd0.png)

instead of this:
![image](https://user-images.githubusercontent.com/5759712/133612472-1f2b352d-53ab-4b1f-a5b0-ed801c2e7b2f.png)

Reply:
Yeah, I got the reset width part.  My point is that in those design system view examples you are setting the border color to lead, but it still black, and that's because the `border:0` is resetting the color. The attribute selector and class selector have the same specifity, but the color one is loaded earlier, so it is overridden by the reset:

![image](https://user-images.githubusercontent.com/68222542/133615665-16a4d9aa-b623-4158-ade8-6bd9de781688.png)

Reply:
fucking hell!
Ok it is changing `border` by `border-width`